### PR TITLE
"Incorporating InnerSource Into Sprints" Outline

### PR DESCRIPTION
**Pull request by Ludmila-N**
solves #350

### CODE REVIEW

File: sprint-implementation/outline.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+### Spring Planning
+- Contributor's point of contact
+    - Their responsibilities
+    - Communicating with Contributor
+
+### During the Sprint
+- Ensure that Contributor's supporter is able to prioritize the support work over team backlog
+- Helpful tips for PR reviews
+    - Remove whitespace discussions from your code reviews - use linters
```
Comment (rrrutledge) [inline sprint-implementation/outline.md:61]: linters and automated tests could also go in pre-requesites.

File: sprint-implementation/outline.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    - Establish rules for contributors and spell them out in CONTRIBUTING.md - see prior section
+- How to organize your backlog
+    - When contribution is more than one story
+        - How to deal with large changes requested by the Contributor's team
+        - What to do if their change has downstream effect
+    - The importance of acceptance criteria
+    - Indications of Contributor's commitment
+    - Managing dependencies
+
+### Spring Planning
```
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply sprint-implementation/outline.md:53]: LOL Sunday night typing... THANK YOU

File: sprint-implementation/outline.md (hunk 1 above)
Comment (Ludmila-N) [reply sprint-implementation/outline.md:61]: I want to separate the suggestions from the true prerequisites. In this case, collaboration is still possible without automated linters, but they will make life oh-so-much-easier. Which is why I put these into tips category.
Do you still think they should be moved into prerequisites, or is this sufficient? We can link into this section from prerequisites article, too.

File: sprint-implementation/outline.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+### During the Sprint
+- Ensure that Contributor's supporter is able to prioritize the support work over team backlog
+- Helpful tips for PR reviews
+    - Remove whitespace discussions from your code reviews - use linters
+    - Implement pre-commit hooks to ensure your tests run before PR is submitted
+    - If you have to explain anything more than once - move that information into your docs
+    - Apply the same diligence in reviewing your team's code as you do with external Contributors
+
+### After the Sprint
```
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply sprint-implementation/outline.md:66]: I'm all for giving thanks, but shouldn't they be bi-directional?
Your sprint guide, by the look of it, assumes that contributors are helping the TC team - whether by fixing bugs or adding attractive features to the product that TC team would like to add but they don't have the capacity right now. 
My top use case is the contributor teams who are willing and ready to contribute but have trouble getting TCs to open up. 
In this case, it's the TC who should be thanked for spending time educating and reviewing code.

[the remark this answers is not in the document]
Comment (Ludmila-N) [reply sprint-implementation/outline.md:66]: In fact, reading what I just wrote, I believe I should add a few more bullets to the outline, specifically explaining the same sections from the contributors perspective, not just TCs. 
So - will add to this PR ASAP 

Thank you for inspiring!

File: sprint-implementation/outline.md (hunk 1 above)
Comment (rrrutledge) [reply sprint-implementation/outline.md:61]: I like splitting out minimal info from deep dive for pre-requisites (and all advice we give).

File: sprint-implementation/outline.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ludmila-N) [reply sprint-implementation/outline.md:74]: @rrrutledge please review - I made changes throughout the outline to address your (and my) thoughts as a result of the discussion above

File: sprint-implementation/outline.md (hunk 1 above)
Comment (Ludmila-N) [reply sprint-implementation/outline.md]: done

File: sprint-implementation/outline.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rrrutledge) [reply sprint-implementation/outline.md:74]: Is there anything specific in the _Contributor_ advice about Sprints?  Or is it just general good _Contributor_ advice?  If the latter, does it overlap with the [_Contributor_ section of the learning path](https://github.com/InnerSourceCommons/InnerSourceLearningPath/tree/master/contributor)?

**Review by rrrutledge — APPROVED** [review]
Everything here does look good, but I am wondering if there is anything specific in the _Contributor_ advice about Sprints?  Or is it just general good _Contributor_ advice?  If the latter, does it overlap with the [_Contributor_ section of the learning path](https://github.com/InnerSourceCommons/InnerSourceLearningPath/tree/master/contributor)?

**Review by lenucksi — COMMENTED** [review]
This looks like great stuff and like a promising video (or set of videos) could arise from it. Thanks for writing it up.

**Pull request merged**